Topics/Tech_Stacks/react_lifecycle.md

```diff
@@ -6,18 +6,23 @@ This documentation provides an overview of React lifecycle methods, explaining w
 
 1. [Introduction](#introduction)
 2. [React Component Lifecycle](#react-component-lifecycle)
-3. [Mounting Phase](#mounting-phase)
-    - [constructor](#constructor)
-    - [render](#render)
-    - [componentDidMount](#componentdidmount)
-4. [Updating Phase](#updating-phase)
-    - [static getDerivedStateFromProps](#static-getderivedstatefromprops)
-    - [shouldComponentUpdate](#shouldcomponentupdate)
-    - [render](#render)
-    - [getSnapshotBeforeUpdate](#getsnapshotbeforeupdate)
-    - [componentDidUpdate](#componentdidupdate)
-5. [Unmounting Phase](#unmounting-phase)
-    - [componentWillUnmount](#componentwillunmount)
+   - [Mounting Phase](#mounting-phase)
+      - [constructor](#constructor)
+      - [render](#render)
+      - [componentDidMount](#componentdidmount)
+   - [Updating Phase](#updating-phase)
+      - [static getDerivedStateFromProps](#static-getderivedstatefromprops)
+      - [shouldComponentUpdate](#shouldcomponentupdate)
+      - [render](#render-1)
+      - [getSnapshotBeforeUpdate](#getsnapshotbeforeupdate)
+      - [componentDidUpdate](#componentdidupdate)
+   - [Unmounting Phase](#unmounting-phase)
+      - [componentWillUnmount](#componentwillunmount)
+   - [Error Handling Phase](#error-handling-phase)
```

Comment:
I believe the Error Handling Phase should be the numbered bullet 3 instead of an unordered bullet. Make sure to change the following with Conclusion (4) and Additional Resources (5).